app/helpers/games_helper.rb

```diff
@@ -1,37 +1,33 @@
 module GamesHelper
   # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity, Metrics/LineLength
   def piece_present?(pieces, game, row_pos, col_pos)
+    white_pieces = {
+      'Pawn'   => '&#9817;',
+      'Rook'   => '&#9814;',
+      'Knight' => '&#9816;',
+      'Bishop' => '&#9815;',
+      'Queen'  => '&#9812;',
+      'King' => '&#9813;'
+    }
+
+    black_pieces = {
+      'Pawn'   => '&#9823;',
+      'Rook'   => '&#9820;',
+      'Knight' => '&#9822;',
+      'Bishop' => '&#9821;',
+      'Queen'  => '&#9818;',
+      'King' => '&#9819;'
```

Comment:
Same error here.  `&#9818;` is Black King and `&#9819;` is Black Queen.  See [this article](https://en.wikipedia.org/wiki/Chess_symbols_in_Unicode) for reference.